scripts/busybox-ping-test/build_aca.sh

```diff
@@ -0,0 +1,160 @@
+#! /bin/sh
+
+# Pull latest ACA code from specified repository, branch and commit
```

Comment:
A general comment: Having a script in Alcor repo for ACA build sounds a bit awkward. Let us plan to migrate those scripts to Alcor-Int repo so that they are independent of component repos like Alcor and ACA.

Reply:
I thought about considerably and made this decision because the current design of the Jenkins job used to run CI doesn't allow more than one repository.

Yes, sometime in near future it can be replaced with "pipeline" and other designs which allow working with multiple repositories.

Reply:
Think about this: Consolidate all scripts in one place, like in Alcor-int, which uses git sub-module (https://git-scm.com/book/en/v2/Git-Tools-Submodules) to point to multiple repos including Alcor controller, ACA and others. We don't need to make the move now but time to plan it :-)

Reply:
I will keep this in mind.